bin/v1/convert_mask.py

```diff
@@ -0,0 +1,79 @@
+                         "The ontological ID for the specific object identified in the mask. In some cases the Mask ID and Object type are the same, for example, if the mask is identifying all cells and each object is an unidentified cell. In this case both the mask ID and object type would be \"CL:0000000\".",
+                         "The X value of the object’s centroid. The top left corner of the image is assumed to have a pixel value of 0. The OME TIFF file XML header must contain the minimum specification as defined below. Specifically, the physical pixel size is necessary to convert pixel coordinates to spatial coordinates. https://docs.google.com/spreadsheets/d/1YnmdTAA0Z9MKN3OjR3Sca8pz-LNQll91wdQoRPSP6Q4/edit#gid=0",
+                         "The Y value of the object’s centroid. The top left corner of the image is assumed to have a pixel value of 0. The OME TIFF file XML header must contain the minimum specification as defined below. Specifically, the physical pixel size is necessary to convert pixel coordinates to spatial coordinates. https://docs.google.com/spreadsheets/d/1YnmdTAA0Z9MKN3OjR3Sca8pz-LNQll91wdQoRPSP6Q4/edit#gid=0",
+                         "The Z value of the object’s centroid. The bottom of the 3D object is assumed to have a pixel value of 0. The OME TIFF file XML header must contain the minimum specification as defined below. Specifically, the physical pixel size is necessary to convert pixel coordinates to spatial coordinates. https://docs.google.com/spreadsheets/d/1YnmdTAA0Z9MKN3OjR3Sca8pz-LNQll91wdQoRPSP6Q4/edit#gid=0",
+                         "The object is located in an anatomical structure with this ontology identifier.", "This column contains the area value of the object.", "This column contains the radius value of the object."])
+        writer.writerow(["Protocol used to derive this feature value (DOI)", "", "", "", "", "", protocol, protocol, protocol, protocol, protocol, protocol, protocol])
+        writer.writerow(["Alternative ID"])
+        writer.writerow(["Requirement level", "REQUIRED", "REQUIRED", "REQUIRED", "REQUIRED", "REQUIRED", "OPTIONAL", "REQUIRED", "REQUIRED", "REQUIRED", "OPTIONAL", "REQUIRED", "REQUIRED"])
+        writer.writerow(["Permissible values", "", "", "", "", "FUSION | Azimuth | STELLAR | In-house | manual | Not applicable"])
+        writer.writerow([""])
+        writer.writerow(["Ojbect ID", "Source file", "Mask name", "Mask ID", "Protocol for mask creation (DOI)", "Annotation tool", "Object type", "x", "y", "z", "Anatomical structure", "Area", "Radius"]) 
+
+    
+        for ome_tiff in ome_tiffs:
+            
+            with tifffile.TiffFile(ome_tiff) as tif:
+                image_data = tif.asarray()
+                ome_metadata = tif.ome_metadata
+
+            #if multiple channels in input, use channel 2 (for testing with existing data)
```

Comment:
In the existing data files channel 2 was the glomerulus mask, but this isn't needed for the pipeline since the output from the previous step will just be the one mask.